app/assets/stylesheets/components/_nav.scss

```diff
@@ -56,54 +57,81 @@ nav {
 		}
 	}
 
-	ul {
+	&__list {
 		padding-inline-start: 0rem;
 		margin: 0rem;
 		width: $nav-width;
-		@include color-and-secondary(background-color, $button-colors);
 		border-right: 1px solid;
-		@include color-and-secondary(border-color, $background-colors);
 		height: 100%;
 		box-sizing: border-box;
+		@include themed(background-color, "background-contrast");
+		@include themed(color, "background");
+		@include themed-with-cards(background-color, "background");
+		@include themed-with-cards(color, "background-contrast");
 	}
 
-	li {
+	&__list-item {
 		list-style: none;
 		font-size: rem(26px);
 		line-height: rem(26px);
 		overflow: hidden;
 		white-space: nowrap;
 
-		&.no-link {
+		&--user-name {
 			color: $color-off-white;
 			font-weight: bold;
 			padding: rem(15px) rem(10px) rem(10px);
 		}
 
-		form>input[type=submit].not-button {
-			background-color: transparent;
-			width: 100%;
-			text-align: inherit;
-		}
-
-		a, span, form>input[type=submit].not-button {
+		&__link {
 			text-decoration: none;
-			@include color-and-secondary(color, $background-colors);
 			display: block;
 			height: 100%;
 			padding: rem(15px) rem(5px) rem(15px) rem(15px);
+			width: 100%;
+			text-align: left;
+			@include themed(color, "background");
+			@include themed-with-cards(color, "cta-bg");
+
+			&--selected {
+				background-color: theme-dark("cta-bg");
+				color: theme-dark("cta-text");
+			}
+
+			&--not-selected:not(:disabled) {
+				&:hover, &:focus {
+					color: theme-dark("cta-text");
+					background-color: rgba(theme-dark("cta-bg"), 0.5);
+				}
+			}
```

Comment:
is there really a need for the `:not(:disabled)`? i see that's being used a few times.

why not set that as the default, and then add a modifier for:
`&:disabled` ?

Reply:
I do this because if I don't, the hover styles apply to disabled buttons too and I have to explicitly and correctly override it for every modifier. Am I missing an obvious way to not have to do that?

Reply:
this is so that I don't have to explicitly override hover styles for every modifier when it's disabled, because I don't want it to provide any interactivity. Is there a simple way to do this that I'm missing? Or is this not the done strategy ux-wise?